assets/component-menu-drawer.css

```diff
@@ -229,12 +229,13 @@ details[open].menu-opening > .menu-drawer__submenu {
   margin-bottom: 2rem;
```

Comment:
When on or both selectors are visible in the drawer, we can remove the `2rem` for better space harmony. [Video](https://screenshot.click/02-34-2uqwa-ah7mj.mp4)

Reply:
✅ Done 🙂